client.lua

```diff
@@ -89,6 +95,10 @@ Citizen.CreateThread(function()
 
 			TriggerEvent('esx_teleportpads:hasExitedMarker', LastPad, LastAction)
 		end
+
+		if not HasAlreadyEnteredMarker then
+            Citizen.Wait(500)
```

Comment:
Uuuum fix this indent

Reply:
let me fix that not sure why the heck it changed that